backend/routes/common_routes/user_routes.py

```
@@ -248,6 +247,17 @@ async def update_user_language(request: Request, data: dict = Body(...)):
         raise
     except Exception as e:
         raise HTTPException(status_code=500, detail=f"Error updating language: {str(e)}")
+
+# Private Router
+@user_private_router.get("/language", response_model=dict)
+async def get_user_language(request: Request):
+    try:
+        language = request.state.user.get("language", "en")
+        if not isinstance(language, str) or not language.strip():
+            language = "en"
+        return {"success": True, "language": language}
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)